libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -2980,6 +2951,71 @@ string YulUtilFunctions::conversionFunction(Type const& _from, Type const& _to)
 	});
 }
 
+string YulUtilFunctions::arrayConversionFunction(ArrayType const& _from, ArrayType const& _to)
+{
+	solUnimplementedAssert(_to.location() != DataLocation::CallData, "Conversion of calldata types not yet implemented.");
```

Comment:
What is to be done here? If "to" is calldata, then "from" also has to be calldata and it even has to be the same type.

Reply:
Hm, that is true.
This assertion was there before this PR. But, in my next PR, I'll change assertion to be different and confirm that `calldata` to `calldata` works fine